.github/actions/unit-tests/action.yaml

```diff
@@ -41,6 +41,19 @@ runs:
       run: tox -e docs
       shell: bash
 
+    - name: Run pip-audit command through GHA
+      # Check dependencies for security vulnerabilities.
+      # Only run on a single python version (the one that does doc builds)
```

Comment:
We have a library or two where we use different versions depending on the python version, right?  Is that likely to be a problem for this?

Reply:
Hmm that's a good call.  I'm just going to ignore docs since that's only used to _test_ the doc builds.  The doc builds are run by readthedocs

Reply:
Actually, I think I'm gonna revert.  We should consider if we want to narrow the versions of Python we support.  A bunch of the libraries we use aren't compatible with older pythons...